services/cloud/cloud.go

```diff
@@ -306,3 +306,26 @@ type EtcdCluster struct {
 	// Tags - The tags attached to the resource group.
 	Tags map[string]*string `json:"tags"`
 }
+
+// AvailabilityZone describes the availabilityZone setting for a virtual machine
+type AvailabilityZone struct {
+	// ID
+	ID *string `json:"ID,omitempty"`
+	// Name
+	Name *string `json:"name,omitempty"`
+	// Type
+	Type *string `json:"type,omitempty"`
```

Comment:
Is Type field required ?

Reply:
type is not used anywhere, so removed it